Hey @Neel-07,

can you run 

```bash
turbo run build --filter=@noterai/web

# or

pnpm frontend:build
```

to test build is passing. Getting error:

```bash
[21:38:28.745] @noterai/web:build: Failed to compile.
[21:38:28.745] @noterai/web:build: 
[21:38:28.745] @noterai/web:build: ./src/components/editor/editor.tsx
[21:38:28.745] @noterai/web:build: Module not found: Can't resolve '@blocknote/mantine'
[21:38:28.745] @noterai/web:build: 
[21:38:28.745] @noterai/web:build: https://nextjs.org/docs/messages/module-not-found
[21:38:28.745] @noterai/web:build: 
[21:38:28.745] @noterai/web:build: Import trace for requested module:
[21:38:28.745] @noterai/web:build: ./src/app/editor/page.tsx
[21:38:28.745] @noterai/web:build: 
[21:38:28.745] @noterai/web:build: ./src/components/editor/editor.tsx
[21:38:28.745] @noterai/web:build: Module not found: Can't resolve '@blocknote/react'
[21:38:28.745] @noterai/web:build: 
[21:38:28.745] @noterai/web:build: https://nextjs.org/docs/messages/module-not-found
[21:38:28.745] @noterai/web:build: 
[21:38:28.745] @noterai/web:build: Import trace for requested module:
[21:38:28.745] @noterai/web:build: ./src/app/editor/page.tsx
[21:38:28.745] @noterai/web:build: 
[21:38:28.746] @noterai/web:build: ./src/components/editor/editor.tsx
[21:38:28.746] @noterai/web:build: Module not found: Can't resolve '@blocknote/core/fonts/inter.css'
[21:38:28.746] @noterai/web:build: 
[21:38:28.746] @noterai/web:build: https://nextjs.org/docs/messages/module-not-found
[21:38:28.746] @noterai/web:build: 
[21:38:28.746] @noterai/web:build: Import trace for requested module:
[21:38:28.746] @noterai/web:build: ./src/app/editor/page.tsx
[21:38:28.746] @noterai/web:build: 
[21:38:28.746] @noterai/web:build: ./src/components/editor/editor.tsx
[21:38:28.746] @noterai/web:build: Module not found: Can't resolve '@blocknote/mantine/style.css'
[21:38:28.746] @noterai/web:build: 
[21:38:28.746] @noterai/web:build: https://nextjs.org/docs/messages/module-not-found
[21:38:28.746] @noterai/web:build: 
[21:38:28.746] @noterai/web:build: Import trace for requested module:
[21:38:28.746] @noterai/web:build: ./src/app/editor/page.tsx
[21:38:28.746] @noterai/web:build: 
[21:38:28.755] @noterai/web:build: 
[21:38:28.755] @noterai/web:build: > Build failed because of webpack errors
[21:38:28.765] @noterai/web:build:  ELIFECYCLE  Command failed with exit code 1.
[21:38:28.765] @noterai/web:build:  WARN   Local package.json exists, but node_modules missing, did you mean to install?
[21:38:28.772] @noterai/web:build: ERROR: command finished with error: command (/vercel/path0/apps/web) /usr/local/bin/pnpm run build exited (1)
[21:38:28.772] @noterai/web#build: command (/vercel/path0/apps/web) /usr/local/bin/pnpm run build exited (1)
[21:38:28.773] 
[21:38:28.773]   Tasks:    0 successful, 1 total
[21:38:28.773]  Cached:    0 cached, 1 total
[21:38:28.773]    Time:    13.62s 
[21:38:28.773] Summary:    /vercel/path0/.turbo/runs/2myoJyjuxF1a6tYi308QDErMxaa.json
[21:38:28.773]  Failed:    @noterai/web#build
[21:38:28.773] 
[21:38:28.775]  ERROR  run failed: command  exited (1)
[21:38:28.791] Error: Command "turbo run build" exited with 1
[21:38:31.280] 
```

--- a/apps/web/package.json
+++ b/apps/web/package.json
@@ -27,7 +27,10 @@
     "react-dom": "^18",
     "socket.io-client": "^4.8.0",
     "tailwind-merge": "^2.4.0",
-    "tailwindcss-animate": "^1.0.7"
+    "tailwindcss-animate": "^1.0.7",
+    "@blocknote/core": "^0.16.0",
+    "@blocknote/mantine": "^0.16.0",
+    "@blocknote/react": "^0.16.0"
   },
   "devDependencies": {
     "@types/node": "^20",

--- a/package.json
+++ b/package.json
@@ -46,10 +46,6 @@
   "homepage": "https://github.com/Himasnhu-AT/noterAI#readme",
   "devDependencies": {
     "turbo": "^2.1.2"
-  },
-  "dependencies": {
-    "@blocknote/core": "^0.16.0",
-    "@blocknote/mantine": "^0.16.0",
-    "@blocknote/react": "^0.16.0"
   }
+  
 }

